CSharp/BatchExplorer/Helpers/ResourceFileStringParser.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.BatchExplorer.Models;
+
+namespace Microsoft.Azure.BatchExplorer.Helpers
+{
+    public static class ResourceFileStringParser
+    {
+        private static readonly string[] FileDelimiter = new[] { ";", Environment.NewLine };
+        private static readonly string[] PartDelimiter = new[] { "=>" };
+
+        public static ResourceFileParseResult Parse(string text)
+        {
+            if (string.IsNullOrWhiteSpace(text))
+            {
+                return ResourceFileParseResult.Empty;
+            }
+
+            var fileStrings = text.Split(FileDelimiter, StringSplitOptions.RemoveEmptyEntries)
+                                  .Select(s => s.Trim())
+                                  .ToList();
+
+            var errors = new List<string>();
+            var files = new List<ResourceFileInfo>();
+
+            foreach (var fileString in fileStrings)
+            {
+                var rawParse = fileString.Split(PartDelimiter, StringSplitOptions.None)
+                                         .Select(s => s.Trim())
+                                         .ToList();
+
+                if (rawParse.Count == 2 && rawParse[0].Length > 0 && rawParse[1].Length > 0)
+                {
+                    files.Add(new ResourceFileInfo(rawParse[0], rawParse[1]));
+                }
+                else
+                {
+                    errors.Add(string.Format("Resource file '{0}' was not of the form 'blobSource => filePath'", fileString));
+                }
+            }
+
+            return new ResourceFileParseResult(errors, files);
+        }
+    }
+
+    public sealed class ResourceFileParseResult
+    {
+        private readonly IReadOnlyList<string> _errors;
```

Comment:
minor: I believe this code uses pascalCase for private member variables, and this.myFoo = * for differentiating myFoo (the private member variable) from myFoo (the method parameter).  It would be good to align with that for consistency.